jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -1,287 +1,70 @@
-
-For example, if we want to connect with Artemis to use the JMS 2.0 spec, we could define our connection factory in this way:
-
-[source,xml,linenums]
-----
-<jms:config name="JMS_Config">
-    <jms:generic-connection specification="JMS_2_0">
-        <jms:connection-factory>
-            <jms:jndi-connection-factory connectionFactoryJndiName="ConnectionFactory" lookupDestination="ALWAYS">
-                <jms:name-resolver-builder
-                        jndiInitialContextFactory="org.apache.activemq.artemis.ActiveMQInitialContextFactory"
-                        jndiProviderUrl="tcp://localhost:61616?broker.persistent=false&amp;broker.useJmx=false">
-                    <jms:provider-properties>
-                        <jms:provider-property key="queue.jndi-queue-in" value="in.queue"/>
-                        <jms:provider-property key="topic.jndi-topic-in" value="in.topic"/>
-                    </jms:provider-properties>
-                </jms:name-resolver-builder>
-            </jms:jndi-connection-factory>
-        </jms:connection-factory>
-    </jms:generic-connection>
-</jms:config>
-----
-
-If you have configured a JNDI context on the connector, you can also look up destinations via JNDI using the `lookupDestination` attribute, which provides three different configurations:
-
-* NEVER: No lookup is done and the destinations are created using the existing JMS session
-* TRY_ALWAYS: First try to find the destination using JNDI, and in case it doesn't exist, create it using the current JMS Session.
-* ALWAYS: If a queue/topic cannot be found via JNDI, fail with a `JMS:DESTINATION_NOT_FOUND` error
-
-For more information regarding the `generic-connection` or `jndi-connection-factory`, see the connector reference docs.
-
-== Set Up the Connection Libraries
+== Prerequisites
 
-NOTE: No matter what type of connection you are using, you always need to configure a library containing the JMS client implementation since the connector is not bound to any particular implementation.
+To use this connector, you must be familiar with:
 
-=== ActiveMQ External Libraries
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
 
-For an ActiveMQ connection, there are three possible libraries that need to be configured depending on your connection settings:
+Before creating an app, you must have access to Anypoint Platform and Anypoint Studio.
 
-* ActiveMQ JMS Client Library:
+== Common Use Cases for the Connector
 
-The `activemq-client` of your choice that provides a valid `org.apache.activemq.ActiveMQConnectionFactory` implementation.
+JMS Connector enables you to:
 
-WARNING: This dependency is always required.
+* Listen for new messages as they arrive to the destination.
+* Publish a new message and send it to the specified destination.
+* Consume a message at any given time in the flow from any given destination.
```

Comment:
I think if you are consuming the message FROM, it should be the source, not the destination?

Reply:
From the source, to the destination...